### DISCUSSION TITLE
UUIDs

### DISCUSSION THREAD

**Discussion by greymda** [alexanderlavrushko/BLE-HUD-navigation-ESP32#13] [General]
probably silly question, but how to you know the UUIDs to use?
is this information available on some resources? 
just common information, if you don't mind sharing.

thank you

**Comment by alexanderlavrushko** [comment]
Those UUIDs in the table on readme page, I've just generated, and then modified one digit, so they're similar. As described here https://www.novelbits.io/uuid-for-custom-services-and-characteristics/

**Reply by greymda** [reply]
oh, i thought Sygic BLE expects specific UUIDs to connect/send data to.

**Reply by alexanderlavrushko** [reply]
The application expects these exact UUIDs, and it's not documented anywhere because it's a prototype feature.

**Reply by greymda** [reply]
yeah, i see. it’s just i really like this project and was wondering if an update of Sygic changes these UUIDs- how to get the new values.

**Reply by alexanderlavrushko** [reply]
I understand. I think Sygic BLE will remain compatible in the nearest future. But what you're saying is a possible situation, because Sygic didn't promise anything about this.
